types/unit/service.pp

```diff
@@ -4,6 +4,7 @@
 #
 type Systemd::Unit::Service = Struct[
   {
+    Optional['AmbientCapabilities']       => Variant[Pattern[/^CAP_[A-Z_]+$/],Array[Pattern[/^CAP_[A-Z_]+$/],1]],
```

Comment:
This doesn't support multiple capabilities separated by whitespace. I thought about that but it became complicated in the template. I don't need it now and if needed you can specify it multiple times. Quoting `man systemd.exec`:
> This option may appear more than once in which case the ambient capability sets are merged.